src/utils/filters/resource-filter-builder/resourceFilterBuilder.js

```diff
@@ -0,0 +1,14 @@
+import dayjs from 'dayjs';
+import { formatDateTimeISO } from '../../time-entry-utils/timeEntryUtils';
+
+export const filterOnOrAfterDate = (resourceDateProperty, date) => {
+  let dateMidnight = formatDateTimeISO(dayjs(date).startOf('day'));
+  return resourceDateProperty + ">='" + dateMidnight + "'";
+};
+
+export const filterBeforeDate = (resourceDateProperty, date) => {
+  let beforeDate = formatDateTimeISO(
```

Comment:
Please can we use `const` when there’s no need to reassign the variable?
```suggestion
  const beforeDate = formatDateTimeISO(
```

Reply:
have now updated